src/test/app/Vault_test.cpp

```diff
@@ -1825,8 +1825,7 @@ class Vault_test : public beast::unit_test::suite
             jvParams[jss::vault] = 42;
             auto jvVault = env.rpc("json", "ledger_entry", to_string(jvParams));
             BEAST_EXPECT(
-                jvVault[jss::result][jss::error].asString() ==
-                "malformedRequest");
+                jvVault[jss::result][jss::error].asString() == "invalidParams");
```

Comment:
Maybe you could use `get_error_info(rpcINVALID_PARAMS).token` here to not duplicate string constant?

Reply:
in unit tests, it is IMO helpful to see actual string.